resources/views/Staff/donations/subscriptions/index.blade.php

```diff
@@ -0,0 +1,228 @@
+@extends('layout.default')
+
+@section('title')
+    <title>Subscriptions - @lang('staff.staff-dashboard') - {{ config('other.title') }}</title>
+@endsection
+
+@section('meta')
+    <meta name="description" content="Subscriptions - @lang('staff.staff-dashboard')">
+@endsection
+
+@section('breadcrumbs')
+    <li class="breadcrumbV2">
+        <a href="{{ route('staff.dashboard.index') }}" class="breadcrumb__link">
+            {{ __('staff.staff-dashboard') }}
+        </a>
+    </li>
+    <li class="breadcrumbV2">
+        <a href="#" class="breadcrumb__link">
+            Donations
+        </a>
+    </li>
+    <li class="breadcrumb--active">
+        Subscriptions
+    </li>
+@endsection
+
+@section('page', 'page__donations-subscriptions--index')
+
+@section('main')
+    <section class="panelV2">
+        <header class="panel__header">
+            <h2 class="panel__heading">Donors</h2>
+            <div class="panel__actions">
+                <div class="panel__action">
+                    <a href="{{ route('staff.donations.subscriptions.create') }}" class="form__button form__button--text">
+                        @lang('common.add') VIP
+                    </a>
+                </div>
+            </div>
+        </header>
+        <div class="panel__body">
+            <!-- Upcoming Users -->
+            <h3>Upcoming</h3>
+            <div class="table-responsive">
+                <table class="table table-condensed table-striped table-bordered">
```

Comment:
Don't bring the bootstrap styles back in here. You can use the moderation page as an example of how the markup should look like.

Reply:
Rgr, will fix it asap.

Reply:
Fixed in e9316451511a17b1f51aa764d477335c596368f9